### ISSUE TITLE
“train_self_model.py” seems to have a lot of bugs.

### ISSUE THREAD

**Issue by odindis** [JasmineRain/NPCNet#1]
I want to try NPCNet, but there are many errors when I run "train_self_model.py".These errors include:
(1) 'NPCNet' object has no attribute 'seg'.
(2) The "final" variable does not contribute the "fine" keyword.
(3) “aux, out, final = net( img ) ” not enough values to unpack.
(4) "stage3, stage4, stage5 = items.values()" not enough values to unpack.
All in all, the program seems to have a lot of bugs. Every time I finish fixing one, another one appears.

Beyond that there are some questions. For example, "loss = aux_loss + seg_loss + point_loss" is replaced by "loss = point_loss + seg_loss". Is "aux_loss" not needed anymore?

**Comment by JasmineRain** [comment]
Thanks for you attention. I've noticed the problem, the bug is caused while doing ablation study, we will fix the bugs and add explainations within this week.
